.azure-pipelines/client.test.live.yml

```diff
@@ -0,0 +1,25 @@
+variables:
+  - group: KeyVault Test Secrets
+  - name: ARM_CLIENTID
+    value: $(java-keyvault-test-arm-client-id)
+  - name: ARM_CLIENTKEY
+    value: $(java-keyvault-test-arm-client-key)
+
+resources:
+- repo: self
+queue:
+  name: Hosted VS2017
+steps:
+- bash: |
+   LOG_PARAMS='-Dorg.slf4j.simpleLogger.defaultLogLevel=error -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn'
+   mvn test -Dhttp.keepAlive=false -Dsurefire.rerunFailingTestsCount=3 $LOG_PARAMS -f pom.client.build.xml 
+  displayName: 'Bash Script'
```

Comment:
Please update this to represent what it is doing, like "Running Live Tests"

Reply:
Updated it.